schemas/JSON/manifests/latest/manifest.installer.latest.json

```diff
@@ -2,778 +2,838 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "description": "A representation of a single-file manifest representing an app installers in the OWC. v1.12.0",
-  "definitions": {
-    "PackageIdentifier": {
-      "type": "string",
-      "pattern": "^[^\\.\\s\\\\/:\\*\\?\"<>\\|\\x01-\\x1f]{1,32}(\\.[^\\.\\s\\\\/:\\*\\?\"<>\\|\\x01-\\x1f]{1,32}){1,7}$",
-      "maxLength": 128,
-      "description": "The package unique identifier"
-    },
-    "PackageVersion": {
-      "type": "string",
-      "pattern": "^[^\\\\/:\\*\\?\"<>\\|\\x01-\\x1f]+$",
-      "maxLength": 128,
-      "description": "The package version"
-    },
-    "Locale": {
-      "type": [ "string", "null" ],
-      "pattern": "^([a-zA-Z]{2,3}|[iI]-[a-zA-Z]+|[xX]-[a-zA-Z]{1,8})(-[a-zA-Z]{1,8})*$",
-      "maxLength": 20,
-      "description": "The installer meta-data locale"
-    },
-    "Channel": {
-      "type": [ "string", "null" ],
-      "minLength": 1,
-      "maxLength": 16,
-      "description": "The distribution channel"
-    },
-    "Platform": {
-      "type": [ "array", "null" ],
-      "items": {
-        "title": "Platform",
-        "type": "string",
-        "enum": [
-          "Windows.Desktop",
-          "Windows.Universal"
-        ]
-      },
-      "maxItems": 2,
-      "uniqueItems": true,
-      "description": "The installer supported operating system"
-    },
-    "MinimumOSVersion": {
-      "type": [ "string", "null" ],
-      "pattern": "^(0|[1-9][0-9]{0,3}|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-4][0-9]{2}|655[0-2][0-9]|6553[0-5])(\\.(0|[1-9][0-9]{0,3}|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-4][0-9]{2}|655[0-2][0-9]|6553[0-5])){0,3}$",
-      "description": "The installer minimum operating system version"
-    },
-    "Url": {
-      "type": [ "string", "null" ],
-      "pattern": "^([Hh][Tt][Tt][Pp][Ss]?)://.+$",
-      "maxLength": 2048,
-      "description": "Url type"
-    },
-    "InstallerType": {
-      "type": [ "string", "null" ],
-      "enum": [
-        "msix",
-        "msi",
-        "appx",
-        "exe",
-        "zip",
-        "inno",
-        "nullsoft",
-        "wix",
-        "burn",
-        "pwa",
-        "portable",
-        "font"
-      ],
-      "description": "Enumeration of supported installer types. InstallerType is required in either root level or individual Installer level"
-    },
-    "NestedInstallerType": {
-      "type": [ "string", "null" ],
-      "enum": [
-        "msix",
-        "msi",
-        "appx",
-        "exe",
-        "inno",
-        "nullsoft",
-        "wix",
-        "burn",
-        "portable",
-        "font"
-      ],
-      "description": "Enumeration of supported nested installer types contained inside an archive file"
-    },
-    "NestedInstallerFiles": {
-      "type": [ "array", "null" ],
-      "items": {
-        "type": "object",
-        "title": "NestedInstallerFile",
-        "properties": {
-          "RelativeFilePath": {
+    "definitions": {
+        "PackageIdentifier": {
```

Comment:
Was this churn intentional?  The PR comment did not mention a refactoring of the schema like this, and I don't understand why we are doing it.

Reply:
Not intentional, will fix